tomviz/python/tomviz/io/_databroker.py

```diff
@@ -1,12 +1,15 @@
+import os
 import numpy as np
-
 import tomviz.utils
 
 from vtk import vtkImageData
 
 _installed = False
+DEFAULT_URL = "https://tiled.nsls2.bnl.gov"
+TILED_URL = os.getenv("TILED_URL", DEFAULT_URL)
```

Comment:
I wonder if we might make beamline settable with env variables like this, but in the C++ is certainly more than enough to work on this tomorrow as we also tie to the andor image

Reply:
Sure, I can do that